src/Chirp.Infrastructure/CheepRepository.cs

```diff
@@ -180,6 +191,7 @@ public async Task Create(CheepCreateDTO cheep)
 
         _db.Add(new Cheep
         {
+            CheepId = new Guid(),
```

Comment:
I think that this is not the way you use Guid. You should use Guid.NewGuid() instead of new Guid().